rocketpy/environment/environment.py

```diff
@@ -688,7 +688,7 @@ def set_date(self, date, timezone="UTC"):
             "Ensemble",
         ]:
             self.set_atmospheric_model(
-                self.atmospheric_model_file, self.atmospheric_model_dict
+                self.atmospheric_model_type, self.atmospheric_model_file, self.atmospheric_model_dict
```

Comment:
My suggestion is for us to prevent the same error from happening again:

```suggestion
                type=self.atmospheric_model_type, file=self.atmospheric_model_file, dictionary=self.atmospheric_model_dict
```

After that, we need to run black again